Header header
int16 fs
float32 accelX
float32 accelY
float32 accelZ
float32 pitch
float32 roll
float32 magX
float32 magY
float32 magZ
float32 gyroX
float32 gyroY
float32 gyroZ
float32 coreTemp
